ckan/logic/action/get.py

```diff
@@ -2542,6 +2547,12 @@ def package_activity_list(context, data_dict):
     # authorized to read.
     data_dict['include_data'] = False
     include_hidden_activity = data_dict.get('include_hidden_activity', False)
+    activity_types = data_dict.pop('activity_types', None)
+    exclude_activity_types = data_dict.pop('exclude_activity_types', None)
+
+    if activity_types is not None and exclude_activity_types is not None:
+        raise ValidationError({'activity_types': ['Cannot be used together with `exclude_filters']})
```

Comment:
exclude_filters?

Reply:
This was thew previous version of the field name
This is proposed to fix @wardi: https://github.com/ckan/ckan/pull/6144